packages/frontend-core/tsconfig.json

```diff
@@ -1,13 +1,12 @@
 {
+  "extends": "../../tsconfig.build.json",
   "compilerOptions": {
     "target": "ESNext",
+    "module": "preserve",
     "moduleResolution": "bundler",
+    "outDir": "./dist",
     "skipLibCheck": true,
-    "paths": {
-      "@budibase/types": ["../types/src"],
-      "@budibase/shared-core": ["../shared-core/src"],
```

Comment:
Will this mess up the import aliases?

Reply:
No, this is now defined in the root `tsconfig.json`